packages/trpc/server/routers/viewer/workflows/update.handler.ts

```diff
@@ -344,6 +366,19 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
               switch (step.action) {
                 case WorkflowActions.EMAIL_HOST:
                   sendTo = [bookingInfo.organizer?.email];
+                  const schedulingType = bookingInfo.eventType.schedulingType;
+                  const hosts = bookingInfo.attendees
+                    .filter((attendee) =>
+                      bookingInfo.eventType.hosts?.some((host) => host.user.email === attendee.email)
+                    )
+                    .map((host) => host.email);
+                  if (
+                    hosts &&
```

Comment:
Initially was looping over hosts and adding fixed ones. But for Round robin with fixed users, fixed users were always made organiser so had to loop over attendees to find lucky user. To keep it same changed to always looping over attendees for hosts as this works for both Collective and Round Robin. 
Lmk if you want to change that.